src/sentry/snuba/metrics/extraction.py

```diff
@@ -1690,9 +1691,11 @@ class SearchQueryConverter:
     The converter can be used exactly once.
     """
 
-    def __init__(self, tokens: Sequence[QueryToken]):
+    def __init__(self, tokens: Sequence[QueryToken], field_mapper: Any = _map_field_name):
```

Comment:
Should the type of `field_mapper` be `Callable[[str], str]`?

Reply:
Good catch, thanks!

Reply:
[727670f](https://github.com/getsentry/sentry/pull/72997/commits/727670f258271bfbc52df750d508c32e3da2edb4)